packages/components/src/internal/components/editable/actions.ts

```diff
@@ -216,10 +216,10 @@ const findLookupValues = async (
 ): ColumnLoaderPromise => {
     const { lookup } = column;
     const { keyColumn } = lookup;
-    const displayColumn = resolveDisplayColumn(column);
+    const displayColumnName = resolveDisplayColumn(column);
 
     const results = await selectRows({
-        columns: [displayColumn, keyColumn],
+        columns: [QueryKey.encodePart(displayColumnName), keyColumn],
```

Comment:
we have been trying hard to remove usages of encodePart from columns recently. Since this is passing the column to the selectRows, can't we just use column.fieldKey?

Reply:
This is where we don't have field key. column.lookup.displayColumn just has `name`. I tried changing `displayColumn` to be fieldKey on server side but worried that would break too many existing usage.